packages/@react-spectrum/badge/stories/Badge.stories.tsx

```diff
@@ -37,7 +37,7 @@ export const Default: BadgeStory = {
 };
 
 export const WithIcon: BadgeStory = {
-  args: {children: <><CheckmarkCircle /><Text>Licensed</Text></>, variant: 'positive'},
+  args: {children: <><CheckmarkCircle /><Text>Licensed</Text></>, variant: 'positive', 'aria-label': 'Done'},
```

Comment:
This is the WithIcon variant, not the IconOnly variant.

`aria-label` does not get added to the Badge component. In order to provide a label for the icon-only variant, we should add `aria-label` to the icon.

```suggestion
  args: {children: <><CheckmarkCircle aria-label="Done" /><Text>Licensed</Text></>, variant: 'positive'},
```

Reply:
Change:
```jsx
export const IconOnly: BadgeStory = {
  args: {children: <CheckmarkCircle />, variant: 'positive', 'aria-label': 'Licensed'},
  name: 'Icon only'
};
```
To:
```jsx
export const IconOnly: BadgeStory = {
  args: {children: <CheckmarkCircle 'aria-label': 'Done: Licensed' />, variant: 'positive'},
  name: 'Icon only'
};
```